staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -278,5 +286,10 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"consistent even when requests arrive at the server before it has been initialized. "+
 		"This option is applicable to Microshift only, this should never be enabled for OCP")
 
+	fs.BoolVar(&s.AnnotateEarlyAndLateRequests, "annotate-early-and-late-requests", s.AnnotateEarlyAndLateRequests, ""+
```

Comment:
what would motivate someone to turn this off or on?

Reply:
most providers don't really care to enable this, I guess. So providing a way enable/disable seems reasonable. If upstream accepts this with a turn off/on switch I am happy to remove it from the upstream pr as well.
(for now I removed it from this pr)